landoui/dockerflow.py

```diff
@@ -16,8 +19,15 @@ def heartbeat():
     and return a 200 iff those services and the app itself are
     performing normally. Return a 5XX if something goes wrong.
     """
-    # TODO check backing services and update tests
-    return '', 200
+    try:
+        response = requests.get(
+            current_app.config['LANDO_API_URL'] + '/__lbheartbeat__'
```

Comment:
Changing to `app.config` might look like more code right now, but if the app grows any more, then keeping `os.environ` isolated will make the code easier to maintain and change.  It made the code in this PR cleaner to write and maintain, and that payoff will happen for any future code, too.